scarb-metadata/tests/scarb_command.rs

```diff
@@ -40,7 +40,7 @@ fn sample_project() {
 
 fn init_project(t: &TempDir) {
     Command::new(scarb_bin())
-        .args(["init", "--name", "hello"])
+        .args(["init", "--name", "hello", "--test-runner", "cairo-test"])
```

Comment:
We have a funny test case here: https://github.com/software-mansion/scarb/blob/a142df1dd521b8a78cd265877d1cd753a7aa437e/.github/workflows/ci.yml#L67-L78
Which runs tests from this package with older version of Scarb, to ensure it's still compatible in regards to metadata format. 
This older version of scarb will not accept `test-runner` as an argument.